### PR TITLE
Add migration guide for astro

### PR DESCRIPTION
**Pull request by panteliselef**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1248/references/astro/migrating-from-astro-community-sdk

### This PR:

-

### CODE REVIEW

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 1]
```diff
@@ -1,5 +1,139 @@
 ---
+title: 'Migrating from the Astro community SDK'
 description: Learn how to migrate from the Astro community SDK to the Clerk Astro SDK.
 ---
 
 # Migrating from the Astro community SDK
+
+In June 2024, we introduced official support for Astro. This minimal guide highlights the breaking changes that were introduced and provides examples on how to resolve them.
```
Comment (royanger) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:8]:
```suggestion
In July 2024, we introduced official support for Astro. This migration guide covers converting from the `astro-clerk-auth` community SDK to the official SDK. It covers the breaking changes that were introduced and provides examples on how to resolve them.
```

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 2]
```diff
@@ -1,5 +1,139 @@
...
+
+
+**Overview:**
+- Install `@clerk/astro`, uninstall `astro-clerk-auth`
+- Update import paths
+- Replace already deprecated APIs
```
Comment (royanger) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:14]:
```suggestion
- Replace deprecated APIs
```

**Review by wobsoriano — APPROVED** [review]
Looking at the community SDK docs, I think you've got everything covered here

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 3]
```diff
@@ -1,5 +1,139 @@
...
+
+    ```bash filename="terminal"
+    pnpm remove astro-clerk-auth
+    pnpm add @clerk/astro
+    ```
+</CodeBlockTabs>
+
+## Update import paths
+We have kept breaking changes to the minimum and there is a great chance that it will not affect your existing codebase.
+As a quick solution find all the usages of `astro-clerk-auth/` and replace them `@clerk/astro/`
```
Comment (royanger) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:39]:
```suggestion
As a quick solution find all the usages of `astro-clerk-auth` and replace them `@clerk/astro`
```
I think this reads a little nicer, with the same effect. Not a required change though.

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 4]
```diff
@@ -1,5 +1,139 @@
...
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
+
+**Bundled** \
+And in order to opt-in to the bundled clerk-js strategy you can do the following
+
+```diff
```
Comment (royanger) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:73]:
```suggestion
```filename="astro.config.mjs" {2-3}
```

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 5]
```diff
@@ -1,5 +1,139 @@
...
+export default defineConfig({
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
+
+### Changes related to stores.
+
+Submodule `/stores` was replaced with `/client/stores`. For the full list of the stores see [here](TODO: add reference)
```
Comment (royanger) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:88]: Should we just create the doc with a title as placeholder, then we can update this link?

**Review by royanger — CHANGES_REQUESTED** [review]

File: docs/references/astro/migrating-from-astro-community-sdk.mdx (hunk 5 above)
Comment (panteliselef) [reply docs/references/astro/migrating-from-astro-community-sdk.mdx:88]: Not sure what you mean here, do you want to drop the todo comment for now ?

Comment (royanger) [reply docs/references/astro/migrating-from-astro-community-sdk.mdx:88]: Sorry. I mean create the `/docs/references/astro/stores` page (or maybe just a section in the Overview depending on how much content there is). Then update the link to point to that page/section.

Comment (panteliselef) [reply docs/references/astro/migrating-from-astro-community-sdk.mdx:88]: Great, that was my intention, glad you are already working on this.

File: docs/references/astro/migrating-from-astro-community-sdk.mdx [hunk 6]
```diff
@@ -1,5 +1,139 @@
...
+```
+
+**Bundled** \
+And in order to opt-in to the bundled clerk-js strategy you can do the following
+
+```filename="astro.config.mjs" {2-3}
+import { defineConfig } from "astro/config";
+import node from "@astrojs/node";
+- import clerk from "astro-clerk-auth";
++ import clerk from "@clerk/astro/bundled";
```
Comment (wobsoriano) [inline docs/references/astro/migrating-from-astro-community-sdk.mdx:77]: I think we forgot to add the `diff` here

**Pull request merged**